biome.jsonc

```diff
@@ -0,0 +1,377 @@
+              }
+            ]
+          }
+        },
+        "useNumberNamespace": "off",
+        "useSingleVarDeclarator": "off",
+        "useTemplate": "off",
+        "noNamespace": "error",
+        "useAsConstAssertion": "error"
+      },
+      "suspicious": {
+        "noAssignInExpressions": "error",
+        "noAsyncPromiseExecutor": "off",
+        "noConfusingVoidType": "off",
+        "noConsoleLog": "error",
+        "noDoubleEquals": "off",
+        "noDuplicateTestHooks": "off",
+        "noExplicitAny": "error",
+        "noExportsInTest": "off",
+        "noFallthroughSwitchClause": "off",
+        "noGlobalIsFinite": "off",
+        "noGlobalIsNan": "off",
+        "noImplicitAnyLet": "off",
+        "noPrototypeBuiltins": "off",
+        "noRedundantUseStrict": "off",
+        "noShadowRestrictedNames": "off",
+        "useDefaultSwitchClauseLast": "off",
+        "useGetterReturn": "off",
+        "noExtraNonNullAssertion": "error",
+        "noMisleadingInstantiator": "error",
+        "noUnsafeDeclarationMerging": "error",
+        "noEmptyBlockStatements": "off" // There is a lot of empty code blocks, should be enabled and clean up separately.
+      },
+      "nursery": {
+        "useConsistentMemberAccessibility": {
+          "level": "error",
+          "options": {
+            "accessibility": "noPublic"
+          }
+        },
+        "noCommonJs": "error",
+        "noRestrictedImports": {
+          "level": "error",
+          "options": {
+            "paths": {
+              "child_process": "Please use node:child_process instead.",
+              "crypto": "Please use node:crypto instead.",
+              "fs": "Please use node:fs instead.",
+              "http": "Please use node:https instead.",
+              "net": "Please use node:net instead.",
+              "os": "Please use node:os instead.",
+              "path": "Please use node:path instead.",
+              "stream": "Please use node:stream instead.",
+              "util": "Please use node:util instead.",
+              "url": "Please use node:url instead.",
+              "worker_threads": "Please use node:worker_threads instead."
+            }
+          }
+        },
+        "noDuplicateElseIf": "error",
+        "noUselessEscapeInRegex": "error",
+        "noIrregularWhitespace": "error",
+        "noOctalEscape": "error",
+        // Need to enable this rule with exception to anonymous functions
+        "useExplicitFunctionReturnType": "off"
+      }
+    }
+  },
+  "javascript": {
+    "formatter": {
+      "jsxQuoteStyle": "double",
+      "quoteProperties": "asNeeded",
+      "trailingCommas": "es5",
+      "semicolons": "always",
+      "arrowParentheses": "always",
+      "bracketSpacing": false,
+      "bracketSameLine": false,
+      "quoteStyle": "double",
+      "attributePosition": "auto",
+      "enabled": true
+    },
+    "linter": {
+      "enabled": true
+    },
+    "globals": ["BigInt"]
+  },
+  "overrides": [
```

Comment:
I tend to think we should not include any override here that is "temporary".
Things like `noConsoleLog: off` in cli is fine, but `noExplicitAny` shouldn't be here.
Instead we should maintain per-line overrides.

I would prefer this PR to NOT change how we manage overrides, rather we just want faster linting.

Reply:
Earlier we used disabling some rules on file level with code comments on top of the file. 

In Biome we can't disable rules at file level with code comments, only way is to add such files in the overrides. Which I believe is better place as it gives more visibility to developers. The code comments just hide from sight once added.